cvat/settings/base.py

```diff
@@ -769,3 +769,15 @@ class CVAT_QUEUES(Enum):
 
 # Indicates the maximum number of days a file or directory is retained in the temporary directory
 TMP_FILE_OR_DIR_RETENTION_DAYS = 3
+
+ABOUT_INFO = {
+    "name": "Computer Vision Annotation Tool",
+    "title": "Open Data Annotation Platform",
+    "description": "CVAT is completely re-designed and re-implemented " +
+                "version of Video Annotation Tool from Irvine, California " +
+                "tool. It is free, online, interactive video and image annotation " +
+                "tool for computer vision. It is being used by our team to " +
+                "annotate million of objects with different properties. Many UI " +
+                "and UX decisions are based on feedbacks from professional data " +
+                "annotation team.",
```

Comment:
I would suggest to remove title and put its content to description

Reply:
This text is still used in 'about' modal window that we have. I agree it may be outdated, but I dont think we want to remove it in terms of this PR.
If we are talking about customization of `/about` endpoint it makes sense so we can edit the description too.